stdlib.go

```diff
@@ -128,24 +141,28 @@ func (a StdlibAdapter) handleMessagePrefix(msg string) string {
 }
 
 const (
-	logRegexpDate = `(?P<date>[0-9]{4}/[0-9]{2}/[0-9]{2})?[ ]?`
-	logRegexpTime = `(?P<time>[0-9]{2}:[0-9]{2}:[0-9]{2}(\.[0-9]+)?)?[ ]?`
-	logRegexpFile = `(?P<file>.+?:[0-9]+)?`
-	logRegexpMsg  = `(: )?(?P<msg>(?s:.*))`
+	StdlibRegexpPatternDate = `(?P<date>[0-9]{4}/[0-9]{2}/[0-9]{2})?[ ]?`
+	StdlibRegexpPatternTime = `(?P<time>[0-9]{2}:[0-9]{2}:[0-9]{2}(\.[0-9]+)?)?[ ]?`
+	StdlibRegexpPatternFile = `(?P<file>.+?:[0-9]+)?`
+	StdlibRegexpPatternMsg  = `(: )?(?P<msg>(?s:.*))`
```

Comment:
Can these be unexported?

Reply:
I see a high benefit of export the regex pattern. library users can re-use them, instead to a copy and own.

What the downside of exporting the strings here?

Reply:
Every exported identifier becomes part of the public API of the package/module, and must be maintained by the maintainers basically forever. I would like to keep the API surface area of this package as small as possible.

Reply:
Alright, I there are not longer exported.